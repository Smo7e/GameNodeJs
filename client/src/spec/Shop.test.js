import React from 'react';
import { render } from '@testing-library/react';
import Shop from '../component/Lobby/component/Shop'; 

jest.mock('../component/Lobby/panel/Panel.tsx', () => () => <div data-testid="mock-panel" />); // Мокируем Panel

describe('Shop Component', () => {
  it('renders correctly', () => {
    const { asFragment } = render(<Shop />);
    expect(asFragment()).toMatchSnapshot();
  });
});
