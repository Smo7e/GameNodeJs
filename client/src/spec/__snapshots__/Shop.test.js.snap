// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shop Component renders correctly 1`] = `
<DocumentFragment>
  <div
    class="baguetteM"
  >
    <div
      class="coin"
    />
    <div
      data-testid="mock-panel"
    />
  </div>
</DocumentFragment>
`;
